lib/parser.js

```diff
@@ -789,6 +818,30 @@ function updateParseArgsForDefaultCapabilities (parser) {
   };
 }
 
+function parseSecurityFeatures (features) {
+  const splitter = (splitOn) => {
+    return (str) => {
+      return str.toString().split(splitOn).map(s => s.trim()).filter(Boolean);
+    };
+  };
+  let parsedFeatures;
+  try {
+    parsedFeatures = splitter(',')(features);
+  } catch (err) {
+    throw new Error('Could not parse value of --allow-insecure. Should be a list of strings separated by commas');
+  }
+
+  if (parsedFeatures.length === 1) {
```

Comment:
&& fs.exists (...)

Reply:
is the call to `exists` more performant than just handling a failed call to `readFile`?

Reply:
it's ok, i like it better because we can give a better error message if the file can't be read.

Reply:
`exists`, actually, verifies whether an item is available for read